package-system/OpenSSL/build_config.json

```diff
@@ -0,0 +1,56 @@
+{
+   "git_url":"https://github.com/openssl/openssl.git",
+   "git_tag":"OpenSSL_1_1_1t",
+   "package_name":"OpenSSL",
+   "package_version":"1.1.1t-rev1",
+   "package_url":"https://github.com/openssl/openssl.git",
+   "package_license":"OpenSSL",
+   "package_license_file":"LICENSE",
+   "cmake_find_target":"FindOpenSSL.cmake",
+   "Platforms":{
+      "Linux":{
+         "Linux":{
+            "cmake_find_template":"FindOpenSSL.cmake.template",
+            "custom_build_cmd": [
+               "./build-linux.sh",
+               "openssl_1_1_1t",
+               "18.04",
```

Comment:
Why is this referencing Ubuntu 18.04?

Reply:
ugh, yes it should. Although it will work using 18.04 as a base, we should be consistent across the board